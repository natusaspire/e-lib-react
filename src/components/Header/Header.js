import React from 'react';

import Logo from '@/components/Logo';
import Navbar from '@/components/Navbar';

const Header = () => (
  <div className="header">
    <div className="header__container header__body">
      <Logo />
      <Navbar />
    </div>
  </div>
);

export default Header;
