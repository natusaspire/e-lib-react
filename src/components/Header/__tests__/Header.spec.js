import React from 'react';
import { shallow } from 'enzyme';

import Header from '@/components/Header';

jest.mock('@/components/Logo', () => 'Logo');
jest.mock('@/components/Navbar', () => 'Navbar');

describe('<Header />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper).toMatchSnapshot();
  });
});
