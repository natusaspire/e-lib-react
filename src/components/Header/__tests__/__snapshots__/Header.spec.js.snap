// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render header content 1`] = `
<div
  className="header"
>
  <div
    className="header__container header__body"
  >
    <Logo />
    <Navbar />
  </div>
</div>
`;
